data.tf

```diff
@@ -1,18 +1,23 @@
 data "aws_region" "current" {}
 data "aws_caller_identity" "current" {}
 
-# Most recent Amazon Linux 2 AMI
-data "aws_ami" "amazon_linux_2" {
+# Most recent Amazon Linux 2023 AMI
+data "aws_ami" "amazon_linux_2023" {
   most_recent = true
   owners      = ["amazon"]
 
   filter {
     name   = "name"
-    values = ["amzn2-ami-hvm*"]
+    values = ["al2023-ami*"]
   }
 
   filter {
     name   = "architecture"
     values = ["x86_64"]
   }
-}
+
+  filter {
+    name   = "virtualization-type"
+    values = ["hvm"]
+  }
+}
```

Comment:
@oycyc can you enable settings in VSCode for auto-last-line

```json
  "files.insertFinalNewline": true,
  "files.trimFinalNewlines": true,
  "files.trimTrailingWhitespace": true,
```

Reply:
I keep forgetting to do this for new installs 😓

Reply:
Yeah... hmm it's there but the last empty line is just not showing up on the GitHub previews, same for the other files:
<img width="444" alt="image" src="https://github.com/masterpointio/terraform-aws-ssm-agent/assets/35899715/2e09b0a0-d51d-4560-aeaf-9ce059ddf05c">
<img width="440" alt="image" src="https://github.com/masterpointio/terraform-aws-ssm-agent/assets/35899715/ff37b839-a24f-4d7d-9ec6-ddfbb79edc43">